File: proga/1_semestr/labs/lab3/src/main/java/dev/lysmux/lab3/common/CanPut.java
package dev.lysmux.lab3.common;

import dev.lysmux.lab3.util.Direction;
import dev.lysmux.lab3.entity.item.Item;
import dev.lysmux.lab3.entity.place.Place;

public interface CanPut {
    String put(Item item, Place place, Direction direction);
}

File: proga/1_semestr/labs/lab3/src/main/kotlin/dev/lysmux/lab3/common/CanPut.kt
package dev.lysmux.lab3.common

import dev.lysmux.lab3.entity.item.Item
import dev.lysmux.lab3.entity.place.Place
import dev.lysmux.lab3.util.Direction

interface CanPut {
    fun put(item: Item, place: Place, direction: Direction): String
}